### PR TITLE
Update stomp_planner_tutorial.rst

### PR DESCRIPTION
**Pull request by mamariomiamo**
### Description

Following current installation commands lead to error. This PR tends to make the installation procedure clearer.

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by LAbron665** [comment]
I have the error below when following the current stomp_planner_tutorial.
image

File: doc/stomp_planner/stomp_planner_tutorial.rst [hunk 1]
```diff
@@ -22,19 +22,20 @@ You should also have gone through the steps in `Visualization with MoveIt RViz P
...
  3. You must also have built `ros-industrial/stomp_ros package <https://github.com/ros-industrial/stomp_ros>`_ and  `ros-industrial/stomp package <https://github.com/ros-industrial/stomp>`_ from source. This needs to be built from source since stomp_ros is not released as a debian yet.
 
 Installing STOMP from Source
 ------------------------------
-As you add and remove packages from your workspace you will need to clean your workspace and re-run the command to install new missing dependencies. Clean your workspace to remove references to the system wide installation of MoveIt: ::
+As you add and remove packages from your workspace you will need to clean your workspace with ``catkin clean`` and re-run the command to install new missing dependencies. Clean your workspace to remove references to the system wide installation of MoveIt: ::
 
-  cd ~/ws_moveit/src
+  cd ~/ws_moveit
   source /opt/ros/melodic/setup.bash
```
Comment (rhaschke) [inline doc/stomp_planner/stomp_planner_tutorial.rst:34]: Shouldn't this tutorial target Noetic?

File: doc/stomp_planner/stomp_planner_tutorial.rst [hunk 2]
```diff
@@ -22,19 +22,20 @@ You should also have gone through the steps in `Visualization with MoveIt RViz P
...
-  wstool init
-  wstool set stomp https://github.com/ros-industrial/stomp_ros.git --git
-  wstool update
+  wstool set -t src stomp_ros https://github.com/ros-industrial/stomp_ros.git --git
+  wstool update -t src stomp_ros
+  wstool merge -t src src/stomp_ros/dependencies.rosinstall
+  wstool update -t src stomp ros_industrial_cke_boilerplate
```
Comment (rhaschke) [inline doc/stomp_planner/stomp_planner_tutorial.rst:38]:
```suggestion
  wstool merge -t src src/stomp_ros/dependencies.rosinstall
  wstool update -t src stomp ros_industrial_cmake_boilerplate
```
Do you really need to build those dependencies from source? Aren't they available as debians? 
If so, use `rosdep` to install them instead.

**Review by rhaschke — COMMENTED** [review]
Generally looks good. Please fix the typo as suggested or - even better - use `rosdep` to install dependencies from debians.

**Comment by rhaschke** [comment]
> I have the error below when following the current stomp_planner_tutorial.

Did you try the instructions suggested in this PR? With those, you should download and build `stomp` itself as well.

File: doc/stomp_planner/stomp_planner_tutorial.rst (hunk 1 above)
Comment (mamariomiamo) [reply doc/stomp_planner/stomp_planner_tutorial.rst]: Yes, it should target Noetic.

File: doc/stomp_planner/stomp_planner_tutorial.rst (hunk 2 above)
Comment (mamariomiamo) [reply doc/stomp_planner/stomp_planner_tutorial.rst]: I have fixed the typo. I think stomp needs to be built from source whereas ros_industrial_cmake_boilerplate could also be installed with apt. Thanks for the advice!

**Review by rhaschke — APPROVED** [review]

**Pull request merged**